plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -352,15 +352,16 @@ def eval_deisolate(self, port_name):
         if not self.dry_run:
             ret = self.ufm_client.deisolate_port(port_name)
             if not ret or ret.status_code != http.HTTPStatus.OK:
-                self.logger.warning("Failed deisolating port: %s with cause: %s... status_code= %s", port_name, self.ports_states[port_name].cause, ret.status_code)        
+                self.logger.warning("Failed deisolating port: %s with cause: %s... status_code= %s",\
+                                     port_name, self.ports_states[port_name].cause, ret.status_code)
                 return
         self.ports_states.pop(port_name)
         log_message = f"Deisolated port: {port_name}. dry_run: {self.dry_run}"
         self.logger.warning(log_message)
         if not self.test_mode:
             self.ufm_client.send_event(log_message, event_id=Constants.EXTERNAL_EVENT_NOTICE, external_event_name="Deisolating Port")
 
-    def get_rate(self, port_obj, counter_name, new_val, timestamp):
+    def get_rate(self,port_obj, counter_name, new_val, timestamp): #pylint: disable=no-self-use
```

Comment:
Please put the disable=XXX in the line prior to the error because it is less visible at the end of the line.